Formula/sonarqube.rb

```diff
@@ -18,6 +18,8 @@ class Sonarqube < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "27759a532884e3d571a16a818f68a6dd7b160f48ea42374716d7248f7ae4fef2"
   end
 
+  # sonarqube ships pre-built x86_64 binaries
+  depends_on arch: :x86_64
```

Comment:
Aren't they universal? The naming suggests so at least.

Reply:
They are not. The audit failure text would be slightly different if they were universal (it would say "unexpected universal binaries" instead of "non-native architecture").

I think the directory structure is just a leftover from when they used to ship 32/64-bit universal binaries.